### PR TITLE
fix(cf): reorg build info details on server group detail and summary

### PR DESCRIPTION
**Pull request by claymccoy**
This adds the artifact version to the server group header, and moves the build info in the server group detail out of the package section.

Screen Shot 2019-05-13 at 3 03 58 PM

### CODE REVIEW

File: app/scripts/modules/cloudfoundry/src/serverGroup/details/sections/BuildSection.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { ICloudFoundryServerGroupDetailsSectionProps } from './ICloudFoundryServerGroupDetailsSectionProps';
+import { CollapsibleSection } from '../../../../../core/src/presentation';
```
Comment (anotherchrisberry) [inline app/scripts/modules/cloudfoundry/src/serverGroup/details/sections/BuildSection.tsx]:
```suggestion
import { CollapsibleSection } from '@spinnaker/core';
```

Comment (claymccoy) [reply app/scripts/modules/cloudfoundry/src/serverGroup/details/sections/BuildSection.tsx]: This causes errors in IJ for some reason. What is the fix for that?

File: app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx [hunk 2]
```diff
@@ -130,6 +131,14 @@ export class SequenceAndBuildAndImages extends React.Component<IServerGroupHeade
           </a>
         )}
         {!!images && <ImageList {...this.props} />}
+        {!!ciBuild && ciBuild.jobUrl && (
+          <span>
+            &nbsp;({ciBuild.version})
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx]: If you are just trying to ensure some whitespace is there, we've been doing `{' '}`

**Comment by claymccoy** [comment]
I'm not a fan for the way that longer strings are represented as values in the detail tab.

File: app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx (hunk 2 above)
Comment (claymccoy) [reply app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx]: okay, I'll fix that too.

**Pull request merged**